static/blockchains/terra/wallets.json

```diff
@@ -0,0 +1,128 @@
+{
+  "terra1g6pwngzfj73wnxm7y5gae0thylphugumc2k48z": {
+    "name": "Hermes Protocol"
+  },
+  "terra15w7ntdhcyfce3twyj5s73gm536q7vn8yf5vp6n": {
+    "name": "Momentum One"
+  },
+  "terra1jv65s3grqf6v6jl3dp4t6c9t9rk99cd8pm7utl": {
+    "name": "Core Distribution"
+  },
+  "terra1fl48vsnmsdzcv85q5d2q4z5ajdha8yu3nln0mh": {
+    "name": "Core Delegation"
+  },
+  "terra1tygms3xhhs3yv487phx3dw4a95jn7t7l8l07dr": {
+    "name": "Core Undelegation & Redelegation"
+  },
+  "terra1ur4davql0wc097frk0fskf6ycce4rwskc49lrr": {
+    "name": "Chinoman10"
+  },
+  "terra15hvzj53lvmtnnj5q8hqruwuff6vp4x77dqmu8c": {
+    "name": "Papi"
+  },
+  "terra18ruqkccl5tp493uhvra0u6jylrzq8t8dv5qs4c": {
+    "name": "Vini"
+  },
+  "terra19sft3gk3enekjkw34zgmnrqq2zcvt2t6ueqhac": {
+    "name": "MC"
+  },
+  "terra17tyjucdrq0qhu3gzxc2u04r5he8je35velus0e": {
+    "name": "Michael"
+  },
+  "terra1e6dvzgn65kalxat5dwapjgg2tyad4gxuvfj3qx": {
+    "name": "Arthur Yeti"
+  },
+  "terra1ky3ayzkeu9ffcm2h452u3tk3d293gjsnu9gvkj": {
+    "name": "GJ"
+  },
+  "terra1gqpvnzggvx4k93s2lrq8fd47pt9jcyf33tdtge": {
+    "name": "Jared"
+  },
+  "terra1scku27w5rsg4392zcaf2y9gnrh0w62t578ez7p": {
+    "name": "Somethingelse"
+  },
+  "terra1ncjg4a59x2pgvqy9qjyqprlj8lrwshm0wleht5": {
+    "name": "Binance Deposit"
+  },
+  "terra19scdgmvhj5rl7f66n06z5pescf0gmeysw6pjrz": {
+    "name": "Binance"
+  },
+  "terra1ksua3uylq7t46kdumhmepywe8u3nglpku8xsk9": {
+    "name": "Binance"
+  },
+  "terra18vnrzlzm2c4xfsx382pj2xndqtt00rvhu24sqe": {
+    "name": "Binance Withdraw"
+  },
+  "terra17m8tkde0mav43ckeehp537rsz4usqx5jayhf08": {
+    "name": "Binance Withdraw"
+  },
+  "terra14l46jrdgdhaw4cejukx50ndp0hss95ekt2kfmw": {
+    "name": "Kucoin Deposit"
+  },
+  "terra1rvxcszyfecrt2v3a7md8p30hvu39kj6xf48w9e": {
+    "name": "Kucoin Withdraw"
+  },
+  "terra1alxk3fmhcl0ga80gl9yj9zp3ezjm4sdh8a9l9c": {
+    "name": "0xPhilipp"
+  },
+  "terra1vzxzwsm0rscya65jg9znu9jls9gtqrv7mc9yh9": {
+    "name": "Rose"
```

Comment:
Although Rose is a known community member, it is also a very generic name, so, similarly to Michael, I'd prefer to remove it from this list.